tests/network/vmi_istio.go

```diff
@@ -496,10 +496,6 @@ var _ = SIGDescribe(" Istio with masquerade binding", decorators.Istio, Serial,
 
 var _ = SIGDescribe(" Istio with passt binding", decorators.Istio, decorators.NetCustomBindingPlugins, Serial, istioTestsWithPasstBinding)
 
-func istioServiceMeshDeployed() bool {
```

Comment:
Isn't this function still used?

Reply:
I removed this change because its not related to the PR.
Sorry for the confusion.